.circleci/config.yml

```diff
@@ -3,22 +3,16 @@ jobs:
   build:
     docker:
       - image: circleci/golang:1.11.5
-    working_directory: /go/src/github.com/student-kyushu/frau
+        environment:
+          GO111MODULE: "ON"
     steps:
       - checkout
       - restore_cache:
           keys:
-            - vendor-{{ checksum "Gopkg.lock" }}
-            - dep
+            - go-mod-v1-{{ checksum "go.sum" }}
```

Comment:
[imo] I often write cache keys in the style below.
`go-mod-{{ checksum "go.sum" }}/1`

Reply:
I adopted your suggestion, thank you.